modules/alloy/src/test/ts/browser/ui/dialog/ModalDialogTest.ts

```diff
@@ -293,7 +293,7 @@ UnitTest.asynctest('ModalDialogTest', (success, failure) => {
       Logger.t(
         'Set the Dialog to "Busy"',
         Step.sync(() => {
-          ModalDialog.setBusy(dialog, (_d, bs) => ({
+          ModalDialog.setBusy(dialog, (bs) => ({
```

Comment:
This is a breaking API change in Alloy. Please either bump to a major version/include a changelog entry or ensure the component is still passed through in this callback.

- [ ] Deal with breaking API change

Reply:
Maybe the `root` element should be passed through here as the first argument, so it's not a breaking change?

Reply:
Swapped back